### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by sbenedicadb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,62 @@
...
+**Open the project**
+
+To open and run the project, open the `code/settings.gradle` file in Android Studio.
+
+**Data Collection mobile property prerequisites**
+
+The test app needs to be configured with the following edge extensions before it can be used:
+- Mobile Core (installed by default)
+- [Identity](https://developer.adobe.com/client-sdks/documentation/identity-for-edge-network/)
+- [Assurance](https://developer.adobe.com/client-sdks/documentation/platform-assurance/)
```
Comment (emdobrin) [inline README.md:32]: let's point to the SDK one here https://developer.adobe.com/client-sdks/documentation/platform-assurance-sdk

File: README.md [hunk 2]
```diff
@@ -1,8 +1,62 @@
...
+$ adb shell am start -W -a  android.intent.action.VIEW -d "audiencetestapp://main?adb_validation_sessionid=ADD_YOUR_SESSION_ID_HERE" com.adobe.audiencetestapp
+```
+
+> Note: replace ADD_YOUR_SESSION_ID_HERE with your Assurance session identifier.
+
+## Related Projects
+
+| Project                                                                              | Description                                                  |
+| ------------------------------------------------------------------------------------ | ------------------------------------------------------------ |
+| [Core Extensions](https://github.com/adobe/aepsdk-core-android)                      | The Core extension represents the foundation of the Adobe Experience Platform SDK. |
```
Comment (emdobrin) [inline README.md:53]: nit: use normal casing here: Core extensions, Assurance extension, AEP SDK sample app for Android

File: README.md [hunk 3]
```diff
@@ -1,8 +1,62 @@
...
+
+### Development
+
+**Open the project**
+
+To open and run the project, open the `code/settings.gradle` file in Android Studio.
+
+**Data Collection mobile property prerequisites**
+
+The test app needs to be configured with the following edge extensions before it can be used:
```
[the remark this answers is not in the document]
Comment (sbenedicadb) [reply README.md:29]: 😬

**Review by emdobrin — APPROVED** [review]

File: Documentation/api-reference.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+#### Kotlin
+
+```kotlin
+Audience.getVisitorProfile { visitorProfile -> 
+    // provide code to process the visitorProfile
+}
+
+```
+
+## registerExtensions
```
[the remark this answers is not in the document]
Comment (emdobrin) [reply Documentation/api-reference.md]: thanks for the update here 💯

**Pull request merged**